library/kong

```diff
@@ -1,11 +1,18 @@
-Maintainers: Mashape Docker Maintainers <support@mashape.com> (@mashape)
-GitRepo: https://github.com/Mashape/docker-kong.git
+Maintainers: Mashape Docker Maintainers <support@konghq.com> (@thekonginc)
+GitRepo: https://github.com/Kong/docker-kong.git
 # needs "Constraints" to match "centos" (which this image is "FROM")
 
-Tags: 0.11-alpine, 0.11.0-alpine
-GitCommit: 55d0b1e894b328483e36b40b178fc080662da773
+Tags: 0.11-alpine, 0.11.1-alpine
+GitCommit: e5bbf3fe757ebe59cfdfda3fade14ef8cd7fa751
 
-Tags: 0.11, 0.11.0, latest
+Tags: 0.11.0-alpine
+GitCommit: b0dfde74dee05124cbdabc0d05ec52eea0b1d058
+
+Tags: 0.11, 0.11.1, latest
+GitCommit: e0d24619ff66cbf432eb687ec642accd2cb5be6c
```

Comment:
This section will need a `GitFetch: refs/heads/centos` in order to get the commit from the other branch.

(we've found it easier to maintain multiple variants by using folders rather than branches, but it is up to you. ex: https://github.com/docker-library/memcached)

Reply:
Ah nice, thanks for the suggestion!